src/ResourceManagement/ApiApp/ApiApp.Tests/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.Resources" version="2.18.0-preview" targetFramework="net45" />
```

Comment:
you don't need to have this, as the test dependency project already wired up for you